src/pages/LendPage/LendDetailPage.tsx

```diff
@@ -0,0 +1,921 @@
+                              >
+                                <CurrencyLogo
+                                  currency={getPoolAssetToken(asset, chainId)}
+                                  size={'36px'}
+                                />
+                                <small className='text-gray29'>
+                                  {asset.underlyingName}
+                                </small>
+                              </Box>
+                            </ItemTableCell>
+                            <ItemTableCell className='poolTableHideCell'>
+                              <p className='caption'>
+                                {convertMantissaToAPR(
+                                  asset.borrowRatePerBlock,
+                                ).toFixed(2)}
+                                %
+                              </p>
+                            </ItemTableCell>
+                            <ItemTableCell>
+                              <small className='text-gray29'>
+                                {midUsdFormatter(asset.borrowBalanceUSD)}
+                              </small>
+                              <p className='caption text-secondary'>
+                                {sdk
+                                  ? convertBNToNumber(
+                                      asset.borrowBalance,
+                                      asset.underlyingDecimals,
+                                    ).toFixed(2)
+                                  : '?'}{' '}
+                                {asset.underlyingSymbol}
+                              </p>
+                            </ItemTableCell>
+                            <ItemTableCell>
+                              <small className='text-gray29'>
+                                {midUsdFormatter(asset.liquidityUSD)}
+                              </small>
+                              <p className='caption text-secondary'>
+                                {sdk
+                                  ? convertBNToNumber(
+                                      asset.liquidity,
+                                      asset.underlyingDecimals,
+                                    ).toFixed(2)
+                                  : '?'}{' '}
+                                {asset.underlyingSymbol}
+                              </p>
+                            </ItemTableCell>
+                          </ItemTableRow>
+                        );
+                      })}
+                    </TableBody>
+                  </Table>
+                </Box>
+              </Box>
+            </Grid>
+            <Grid item xs={12} sm={12} md={6}>
+              <Box className='poolDetailsItemWrapper'>
+                <Box className='poolDetailsItemTop'>
+                  <h6>{t('poolInfo')}</h6>
+                </Box>
+                <Box display={'flex'} pb={'16px'} flexDirection={'column'}>
+                  <Box className='poolDetailsInfoRow'>

```

Comment:
This can be condensed and the template can be mapped from an array of items.

as in the comment above